### PR TITLE
PS-7311 Mention supported Collation for RocksDB in documentation (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
Merged with 5.7
modified myrocks/limitations

### CODE REVIEW

**Comment by inikep** [comment]
These changes will not be correct in 8.0.23 after https://github.com/percona/percona-server/pull/4136 will be merged.

File: doc/source/myrocks/limitations.rst [hunk 1]
```diff
@@ -23,7 +23,8 @@ The MyRocks storage engine lacks the following features compared to InnoDB:
 You should also consider the following:
 
 * :file:`*_bin` (e.g. ``latin1_bin``) or binary collation should be used
-  on ``CHAR`` and ``VARCHAR`` indexed columns.
+  on ``CHAR`` and ``VARCHAR`` indexed columns. The following binary collations
+  are supported: ``binary``, ``latin1_bin``, and ``utf8_bin``.
```
Comment (inikep) [inline doc/source/myrocks/limitations.rst]: This is true for PS-5.7.
From PS-8.0.23 all collations are now supported for `CHAR` and `VARCHAR` indexed columns.
Implemented with https://jira.percona.com/browse/PS-7310

Comment (patrickbirch) [reply doc/source/myrocks/limitations.rst]: Removed the binary columns sentence.

**Review by inikep — APPROVED** [review]
LGTM

**Comment by patrickbirch** [comment]
Closed and moved to the documentation repository
https://github.com/percona/psmysql-docs/pull/33

**Pull request closed without merging**